### ISSUE TITLE
Add LOGSFP to METGRID.TBL.ARW

### ISSUE THREAD

**Issue by dmey** [wrf-model/WPS#115]
Running `metgrid.exe` with data containing the log surface pressure (e.g. ECMWF hybrid level data) results in the following warning:

```
WARNING: Entry in METGRID.TBL not found for field LOGSFP. Default options will be used for this field!
```

**Comment by weiwangncar** [comment]
The reason for this is that this field, LOGSFP, isn't a required field for
the WRF model. What do you need this field for?

**Comment by dmey** [comment]
When using ECMWF hybrid model level data, the corresponding VTable includes it -- see:

https://github.com/wrf-model/WPS/blob/23bd63af662609cb96168f0c07544a33af19d07c/ungrib/Variable_Tables/Vtable.ECMWF_sigma#L8

**Comment by mgduda** [comment]
I don't think there should be a problem to include an entry for `LOGSFP`. If I have a few minutes today or tomorrow, I'll put together a PR.

**Comment by weiwangncar** [comment]
LOGSFP is used to compute 3D pressure, and then height and RH when using
the utility program, calc_ecmwf_p.exe. Once the these fields are computed,
the metgrid program will proceed to process all the relevant fields
required by the model. LOGSFP isn't one of them.

**Comment by dmey** [comment]
@weiwangncar in that case I think that warning is misleading. If `LOGSFP` is not handled by `metgird.exe` under any circumstances, the warning should never be raised. Are there any other variables that should be ignored by `metgird.exe`?

**Comment by weiwangncar** [comment]
If LOGSFP is in the intermediate formatted file, then metgrid.exe will
interpolate it. If the warning message causes concern, we can provide a fix
to deal with it, as suggested by Michael. We probably don't have any other
field that looks like this.

**Comment by mgduda** [comment]
PR #116 should address this issue.

**Comment by mgduda** [comment]
Fixed by PR #116 .
